client/src/js/samples/components/Create/__tests__/ReadSelector.test.js

```diff
@@ -1,4 +1,5 @@
-import ReadSelector from "../ReadSelector";
+import { Button, SearchInput } from "../../../../base";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Button' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5211749)